src/SFA.DAS.EmployerAccounts.UnitTests/Queries/GetAccountPAYESchemes/WhenIGetAccountPayeSchemes.cs

```diff
@@ -48,31 +46,25 @@ public void Arrange()
                 Amount = 0.5m
             };
 
+            _payeView.EnglishFraction = _englishFraction;
```

Comment:
_englishFraction isn't needed in the test any more as you're just doing an object comparison with whatever has been returned from the new service

Reply:
I just left it as it was part of the object being compared, but not necessary agreed. Removed.